src/pytest_mock_resources/fixture/database/redis.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from pytest_mock_resources.compat import redis
+from pytest_mock_resources.container import get_docker_host
+from pytest_mock_resources.container.redis import redis_config
+from pytest_mock_resources.fixture.database.generic import assign_fixture_credentials
+
+
+def create_redis_fixture(**kwargs):
+    """Return a pytest fixture representing a Redis client.
+
+    :param scope: The scope of the fixture can be specified by the user, defaults to "function".
```

Comment:
google docstring style please! if we dont have `"sphinx.ext.napoleon",` enabled in conf.py, we'd need to, but it ought to be preinstalled these days